docs/source/eventing/actions/label.rst

```diff
@@ -20,17 +20,25 @@ Changes the text and color of a generic label device.
 +----------+------------------+---------------------+---------------------------------------------------+
 | TO       | |yes|            | |no|                | New text                                          |
 +----------+------------------+---------------------+---------------------------------------------------+
-| COLOR    | |no|             | |no|                | New color. If omitted, color is set to black"     |
-+----------+------------------+---------------------+---------------------------------------------------+
 | **Name** | **Required**     | **Multiple Values** | **Description**                                   |
 +----------+------------------+---------------------+---------------------------------------------------+
+| COLOR    | |no|             | |no|                | New color                                         |
++----------+------------------+---------------------+---------------------------------------------------+
+| BLINK    | |no|             | |no|                | Blinking on or off                                |
++----------+------------------+---------------------+---------------------------------------------------+
 | FOR      | |no|             | |no|                | | Determine how long this new label lasts         |
 |          |                  |                     | | before we change back to the previous label     |
 +----------+------------------+---------------------+---------------------------------------------------+
 | AFTER    | |no|             | |no|                | After how long do we want the new label to be set |
 +----------+------------------+---------------------+---------------------------------------------------+
 
-.. note:: Units for ``FOR`` and ``AFTER``
+.. note:: 1. Valid values for ``BLINK``
```

Comment:
I don't like the bulleted notes. Also, did you test this yourself?

Reply:
Fixed and yes, I tested it but apparently overlooked it.